packages/domain/src/use-cases/transactions/__tests__/PayTransaction.spec.ts

```diff
@@ -51,16 +51,15 @@ describe(`Given the ${PayTransaction.name} interactor`, () => {
     it(`should:
         - create an ${UnsignedTransaction.name}<T>
         - sign and broadcast the transaction with the user's wallet
-        - queue the resulting ${NativeTransaction.name} into the ${TransactionQueue.name}
-        - present successful result`, async () => {
+        - queue the resulting ${NativeTransaction.name} into the ${TransactionQueue.name}`, async () => {
       const wallet = mockWallet();
       const queue = mockTransactionQueue<AnyTransactionRequestModel>();
       const gateway = mockIPayTransactionGateway({
         request,
         wallet,
         unsignedTransaction,
       });
-      const presenter = mock<IPayTransactionPresenter>();
+      const presenter = mock<IPayTransactionPresenter<AnyTransactionRequestModel>>();
```

Comment:
Unrelated to these changes but the first time i've seen `PayTransaction` and it seems to not read correct. 

Would `PaidTransaction` be more appropriate?

Reply:
In general use cases are actions that can be performed on a given app. So the verb is usually preferable over nouns.

This one being about "paying for a transaction"... the "paid transaction" could describe a good name for an outcome objects.

Maybe "pay transaction" could be "pay for transactions"? Not sure.